lib/java/ydoc-server/src/main/java/org/enso/ydoc/server/Main.java

```diff
@@ -54,10 +57,18 @@ public static AutoCloseable launch(String ydocHost, String ydocPort, MessageCall
       if (callbacks != null) {
         builder.callbacks(callbacks);
       }
+      var hostAccess =
+          WebEnvironment.defaultHostAccess
+              // allowImplementations is required to call methods on JS objects from Java, i.e. to
+              // call methods on `YjsChannel` object returned from JS
+              .allowImplementations(YjsChannel.class)
```

Comment:
- allowing implementation of a well known interface is OK
- but opening up `allowAccess` to a random `callbacks.getClass()` class is a security flaw
- please create a proxy around `callbacks` and expose its methods 
   - preferrably via `@HostAccess.Explicit`
   - or via `allowAccess(ProxyMessageCallbacks.getDeclararedMethod(""))` but while hardcoding `ProxyMessageCallbacks` class